src/utils/walletExtensionV0.ts

```diff
@@ -80,12 +80,29 @@ export interface WalletExtensionV0State {
   }
   localStorage: {
     ADDRESS_BOOK_CONFIG: StringifiedType<Array<{ name: string; address: `oasis1${string}` }>>
-    LANGUAGE_CONFIG: undefined | 'en' | 'zh_CN' // Ignored in migration
+    LANGUAGE_CONFIG: undefined | 'en' | 'zh_CN'
     NETWORK_CONFIG: undefined | any // Ignored in migration
     DISMISSED_NEW_EXTENSION_WARNING: undefined | any // Ignored in migration
   }
 }
 
+export interface MigratingV0State {
+  mnemonic: string
+  invalidPrivateKeys: { privateKeyWithTypos: string; name: string; address: `oasis1${string}` }[]
+  language: LanguageKey
+  state: PersistedRootState
+}
+
+export async function deleteV0ExtProfile() {
+  if (runtimeIs !== 'extension') throw new Error('Can only delete V0 profile in an extension')
+  const browser = await import('webextension-polyfill')
+  await browser.storage.local.remove('keyringData')
```

Comment:
One thing that I would immediately suggest, let's not delete the migrated encrypted data. Let's save it under different key, `keyringDataMigrated` or something similar. In case there is a migration edge case we have not foreseen, we can always push a new version to fix it. And after certain time period(1-3 months), push a new version, that would clean up the migration data. The data is encoded, it shouldn't pose any security risk, it should only benefit us.